### ISSUE TITLE
Problemas com a passagem de fotos

### ISSUE THREAD

**Issue by Whemerson** [elementary/photos#547]
Quando eu  passo as minhas fotos no app elas ficam mescladas, ou sejam, se eu passa de uma foto para a outro a anterior se mecla com a seguinte e não sai.

**Comment by ryonakano** [comment]
Translated using Google Translation:

> When I pass my photos on the app they are mixed, that is, if I move from one photo to another, the previous one will move with the next one and it won't come out.

**Comment by ryonakano** [comment]
@Whemerson Could you attach a screenshot that describes your issue?

Also, I have one question: do you use a custom theme, for example, to make elementary OS look like macOS even more?

**Comment by Whemerson** [comment]
Não, fiz uma instalação recente para o elentary os 5.1 e anida não fiz costumizações

**Comment by ryonakano** [comment]
> No, I did a recent installation for elementary os 5.1 and anida did not make customizations
> (Google Translation)

Thank you for your answer. Would it be possible for you to take screenshot(s) of your issue so that we can share your issue visually?

**Comment by RassielMex** [comment]
Hello, 
Same issue but in this case im using a gtk theme. If I unenable it, problem is solved. 
![Screenshot from 2020-11-02 15 24 19](https://user-images.githubusercontent.com/68927038/97920746-86576c00-1d1f-11eb-9d1c-7509a14fb210.png)
